problems/4.arrays.js

```diff
@@ -12,15 +16,26 @@ function firstAndLastAverage() {
 * @param {number[]} arr - The input array
 * @returns {number[]} - The duplicated array
 */
-function duplicateArray() {
+function duplicateArray(array) {
+  array = []
+  arraysCopy = [];
+  i = -1;
+  while (++i < array.length){
+    arrayCopy[i] = array[i];
+  }
+
 
 }
 
 /** Removes the first 3 elements from an array. If the array has fewer than 3 elements, return an empty array.
 * @param {any[]} arr - The input array
 * @returns {any[]} - The array with the first 3 elements removed
 */
-function removeFirstThreeElements() {
+function removeFirstThreeElements(array) {
+  let array = array.length
+  return [arry.shift[0,1,2]];
```

Comment:
shift() is  a method that is invoked to remove the first element. Does not take in arguments. When you're unsure of a method use, look it up on MDN. 
YOu need to return the array after the first 3 elements have been removed.